1_fetch/src/fetch_sb_data.R

```diff
@@ -0,0 +1,19 @@
+download_sb_file <- function(sb_id, file_name, out_dir){
+  #'
+  #' @description Function to download file from ScienceBase 
+  #'
+  #' @param sb_id string - the id of the science base item
+  #' @param file_name string - the name of the file in the science base item to download
+  #' @param out_dir string - the directory where you want the file downloaded to
+  #'
+  #' @value string the out_path
+
+  out_path = file.path(out_dir, file_name)
+  # Get the data from ScienceBase:
+  sbtools::item_file_download(sb_id = sb_id,
+                              names = c(file_name),
+                              destinations = c(out_path),
```

Comment:
I wondered why the `c()` was used here. I didn't realize (or had forgotten) that you could input a string vector to only return file names of interest. That might come in handy for downloading the network adjacency matrix and not the other network attribute files associated with the same data release and `sb_id`.

Reply:
I put the `c()`  because in the [documentation ](https://rdrr.io/github/USGS-R/sbtools/man/item_file_download.html) it says it's a string _vector_. So I think you can pass multiple file names which is what I was thinking. But I tried without the `c()` and it worked. So I'll take those out.

Reply:
@jsadler2 Just to clarify, I don't think you should necessarily take those out. I was just commenting that I didn't know you could do that (but sure enough, it's clear in the documentation) - good to know about!